ntbs-ui-tests/start_selenium.ps1

```diff
@@ -1,5 +1,17 @@
 #!/usr/local/bin/powershell
 
-.\node_modules\.bin\selenium-standalone.ps1 install
+ping 127.0.0.1
 
-.\node_modules\.bin\selenium-standalone.ps1 start
+telnet host 4444
+netstat -an
+
+.\node_modules\.bin\selenium-standalone.ps1 install --drivers.chrome.version=89.0.4389.23
+
+netstat -an
+
+try {
+    .\node_modules\.bin\selenium-standalone.ps1 start --drivers.chrome.version=89.0.4389.23
```

Comment:
Is there a reason why there is a specific version of Chrome here? Locking ourselves into a single version doesn't seem ideal, and maintaining this for every Chrome release also feels like a pain

Reply:
This is a great question! 

The situation here is:
* `selenium-standalone` installs the latest version of `chromedriver` by default.
* `chromedriver` refuses to run against versions of `chrome` that are lower than its own.
* Github runners run updates once per week (they have a lot of software, and apparently the update process often takes several days!)

So without fixing the version, everytime `chromedriver` and `chrome` update at the same time (they are independent, but I imagine they mostly stay in tandem) it's likely that there'll be a lag of a few days where the tests will fail because the VM running the tests doesn't have the latest version of `chrome`.

Now that I've written this down, I've remembered that I didn't end up using these powershell scripts, so we can remove this here... but the same problem happens in the workflow script. We could potentially try and do something like:
* Check which version of `chrome` is installed
* Try and use the corresponding version of `chromedriver`
I'm not sure how tricky this would be... do their version numbers line up exactly? If we wrote `--drivers.chrome.version=89` would the above work?

I think we should defer this to a tech debt ticket.